README.md

```diff
@@ -17,10 +17,12 @@ Hopefully this will help you to get up and running with Sitecore and Docker. By
 As Sitecore does not distribute Docker images, the first step is to build the required Docker images.
 
 ## Pre-build steps
-For this you need the Sitecore installation files and a Sitecore license file. 
-What files to use are set in the [build configuration](./build/Build.cs).
+For this you need to place the Sitecore installation files and a Sitecore license file in the `files` directory. Which files to use are defined in the build configuration files:
+- [XP build config](./build/Build.Xp.cs)
+- [XC build config](./build/Build.Xc.cs)
+- [Overall build config](./build/Build.cs)
 
-The xp0 Sitecore topology requires SSL between the services, for this we need self signed certificates for the 
+The XP0 Sitecore topology requires SSL between the services, for this we need self signed certificates for the 
 xConnect and SOLR roles. You can generate these by running the `./Generate-Certificates.ps1` script (note that this requires an Administrator elevated powershell environment and you may need to set the correct execution policy, e.g. `PS> powershell.exe -ExecutionPolicy Unrestricted`).
```

Comment:
Moeten we die GenerateCertificates neit als onderdeel van de build doen?

Reply:
https://github.com/avivasolutionsnl/sitecore-docker/issues/14#issuecomment-457239065